spec/decorators/case/sar/offender_complaint_decorator_spec.rb

```diff
@@ -92,13 +93,19 @@
   end
 
   describe "#highlight_flag" do
-    it "returns blank string when priority is normal" do
-      expect(offender_sar_complaint.highlight_flag).to eq ""
+    context "when Offender SAR Complaint - high priority" do
+      it 'returns string of "High priority"' do
+        priority_case = create(:accepted_complaint_case, :high_priority, :flagged).decorate
+        expect(priority_case.highlight_flag).to eq '<div class="offender_sar_complaint-priority_flag">' \
+          '<span class="visually-hidden">This is a </span>High priority<span class="visually-hidden"> case</span></div>'
+      end
     end
 
-    it 'returns string of "High" when priority is high' do
-      offender_sar_complaint.priority = "high"
```

Comment:
The existing tests just update the priority value in place, so this might be a simpler way to do things rather than creating a new trait in the factory.